images/router/nginx/Dockerfile

```diff
@@ -5,19 +5,20 @@
 #
 FROM openshift/origin
 
-RUN INSTALL_PKGS="nginx" && \
-    yum install -y "epel-release" && \
-    yum install -y $INSTALL_PKGS && \
-    rpm -V $INSTALL_PKGS && \
+ENV NGINX_VERSION 1.13.8-1.el7_4.ngx
+
+RUN echo -e "[nginx]\nname=nginx repo\n" >> /etc/yum.repos.d/nginx.repo && \
+    echo "baseurl=http://nginx.org/packages/mainline/centos/7/\$basearch/" >> /etc/yum.repos.d/nginx.repo && \
+    echo -e "gpgcheck=0\nenabled=1" >> /etc/yum.repos.d/nginx.repo && \
```

Comment:
@ramr `echo "baseurl=http://nginx.org/packages/mainline/centos/7/\$basearch/"` is dynamic. Considering that, is it OK to leave the whole thing in the Dockerfile?

Reply:
@pleshakov  what I meant by static was that this info is not really changing as far as the dockerfile is concerned. 
The file itself would still have "$basesearch" inside it - but that is expanded at `run` time rather than in the dockerfile here.
Maybe just put this snippet into a file and then append it to an existing file (assuming that `/etc/yum.repos.d/nginx.repo` already exists). 
Something like:
```
COPY ./<path-to>/nginx.repo /tmp/nginx.repo.addition
...
RUN cat /tmp/nginx.repo.addition >> /etc/yum.repos.d/nginx.repo
```

Or is this just creating/writing it? In that case, maybe even: 
```COPY ./<path-to>/nginx.repo /etc/yum.repos.d/```  
 if that works.